pep-9999.rst

```diff
@@ -354,6 +354,46 @@ specified in ``__all__`` are imported::
   public_attr: int
   _private_looking_public_attr: int
   private_attr: int
+  
+Enums
+-----
+
+Enum classes are supported in stubs, regardless of the Python version targeted by
+the stubs.
+
+Enum members may be specified just like other forms of assignments, for example as
+``x: int``, ``x = 0``, or ``x = ...``.  The first syntax is preferred because it
+allows type checkers to correctly type the ``.value`` attribute of enum members,
+without providing unnecessary information like the runtime value of the enum member.
+
+Additional properties on enum members should be specified with ``@property``, so they
+do not get interpreted by type checkers as enum members.
+
+Yes::
+
+    from enum import Enum
+    
+    class Color(Enum):
+        RED: int
+        BLUE: int
+        @property
+        def rgb_value(self) -> int: ...
+
+    class Color(Enum):
```

Comment:
should this be in the "No" section?

Reply:
I intended it to be *allowed* (until recently this is how most enums in typeshed were defined) but *discouraged* (because it doesn't allow as precise type checking). Maybe part of this section should be moved into the style guide.

Reply:
Ah, I see. Yeah, maybe this should be moved into the style section then. It seems sort of weird to have one discouraged example in "Yes" and one in "No", when I assume both of them would parse just fine.